src/main/webapp/WEB-INF/components/footer.html

```diff
@@ -1,38 +1,36 @@
 <footer class="footer text-center">
   <div class="container">
     <div class="row">
-      <!-- Footer Location-->
-      <div class="col-lg-4 mb-5 mb-lg-0">
-        <h4 class="text-uppercase mb-4">Location</h4>
-        <p class="lead mb-0">
-          2215 John Daniel Drive
-          <br />
-          Clark, MO 65243
-        </p>
-      </div>
-      <!-- Footer Social Icons-->
-      <div class="col-lg-4 mb-5 mb-lg-0">
-        <h4 class="text-uppercase mb-4">Around the Web</h4>
-        <a class="btn btn-outline-light btn-social mx-1" href="#!"><i class="fab fa-fw fa-facebook-f"></i></a>
-        <a class="btn btn-outline-light btn-social mx-1" href="#!"><i class="fab fa-fw fa-twitter"></i></a>
-        <a class="btn btn-outline-light btn-social mx-1" href="#!"><i class="fab fa-fw fa-linkedin-in"></i></a>
-        <a class="btn btn-outline-light btn-social mx-1" href="#!"><i class="fab fa-fw fa-dribbble"></i></a>
+      <div class="col-lg-8 mb-5 mb-lg-0">
+        <h4 class="text-uppercase mb-4">About us</h4>
+        <div class="row">
+          <div class="col-lg-4">
```

Comment:
```suggestion
          <div class="col-lg-4" mb-3 mb-lg-0>
```
So it doesn't look to cramped on small screens. Same for the other 2 divs.